Wikipedia/Code/IconBarButtonItem.swift

```diff
@@ -12,8 +14,9 @@ class IconBarButtonItem: UIBarButtonItem {
         let customView = UIButton(type: .system)
         customView.setImage(image, for: .normal)
         customView.addTarget(target, action: action, for: controlEvents)
-        customView.adjustsImageWhenDisabled = false
-        if iconInsets != .zero { customView.imageEdgeInsets = iconInsets }
+        var deprecatedCustomView = customView as DeprecatedButton
```

Comment:
Looks like the `customView` below needs this `as DeprecatedButton` sorcery as well. (Sorry GitHub wasn't letting me comment on that line)

Reply:
Fixed! 🧙